webapi/__init__.py

```diff
@@ -1,27 +1,32 @@
-VERSION = (0, 3, 2)
+from __future__ import unicode_literals
+
+VERSION = (0, 4, 3)
 
 
 try:
     from deluge.plugins.init import PluginInitBase
 
     class CorePlugin(PluginInitBase):
-
         def __init__(self, plugin_name):
-            from core import Core as _plugin_cls
+            from .core import Core as _plugin_cls
             self._plugin_cls = _plugin_cls
             super(CorePlugin, self).__init__(plugin_name)
 
-
     class GtkUIPlugin(PluginInitBase):
```

Comment:
This can be removed entirely due to Gtk 3.

Reply:
Will do.